design/shop_manager_class_design.md

```diff
@@ -0,0 +1,236 @@
+shop_manager 
+
```

Comment:
Instead of having a third table (orders_by_items), you could add item_id as a column to the orders table as a foreign key, to have a one-to-many relationship between the items and orders table.